packages/features/ee/teams/components/DisableTeamImpersonation.tsx

```diff
@@ -40,7 +40,6 @@ const DisableTeamImpersonation = ({
           setAllowImpersonation(_allowImpersonation);
           mutation.mutate({ teamId, memberId, disableImpersonation: !_allowImpersonation });
         }}
-        switchContainerClassName="mt-6"
       />
```

Comment:
Was it a UI bug?

Reply:
No, i had to move this component above the member's list because user had to scroll a lot to find this setting

Reply:
Let's keep unrelated changes in separate PRs.

Reply:
It's related to this PR. As I have implemented lazy loading when you scroll to the end of members list you would see two toggle buttons but as soon as new page is loaded this toggle is again moved at the end. so user would have to scroll and load all the team members just to change the toggle button